pkg/install/storage.go

```diff
@@ -39,49 +39,32 @@ func (i *Installer) createDNS(ctx context.Context) error {
 	return i.dns.Create(ctx, i.doc.OpenShiftCluster)
 }
 
-func (i *Installer) installStorage(ctx context.Context, installConfig *installconfig.InstallConfig, platformCreds *installconfig.PlatformCreds, image *releaseimage.Image) error {
-	clusterID := &installconfig.ClusterID{
-		UUID:    uuid.NewV4().String(),
-		InfraID: "aro",
-	}
-
-	g := graph{
-		reflect.TypeOf(installConfig): installConfig,
-		reflect.TypeOf(platformCreds): platformCreds,
-		reflect.TypeOf(image):         image,
-		reflect.TypeOf(clusterID):     clusterID,
-	}
-
-	i.log.Print("resolving graph")
-	for _, a := range targets.Cluster {
-		_, err := g.resolve(a)
-		if err != nil {
-			return err
-		}
-	}
-
-	resourceGroup := stringutils.LastTokenByte(i.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID, '/')
-
+func (i *Installer) createResourceGroup(ctx context.Context, installConfig *installconfig.InstallConfig) error {
 	i.log.Print("creating resource group")
+	rgName := stringutils.LastTokenByte(i.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID, '/')
 	group := mgmtresources.Group{
 		Location:  &installConfig.Config.Azure.Region,
 		ManagedBy: to.StringPtr(i.doc.OpenShiftCluster.ID),
 	}
 	if _, ok := i.env.(env.Dev); ok {
 		group.ManagedBy = nil
 	}
-	_, err := i.groups.CreateOrUpdate(ctx, resourceGroup, group)
+	_, err := i.groups.CreateOrUpdate(ctx, rgName, group)
```

Comment:
Using the name `rgName` here to contrast it with the variable `group`. Also, the parameter taken by `CreateOrUpdate` is `resourceGroupName`, so it seemed like a good choice.

Changed it to `rgName` elsewhere as well to be consistent.

Reply:
(Could just call it `resourceGroupName` instead.)